### ISSUE TITLE
Position inversion

### ISSUE THREAD

**Issue by joe-sydney** [sillyfrog/Automate-Pulse-v2#11]
Hey sillyfrog, thanks for this great integration.

I have a feature request - at the moment, closed is 0% and open is 100%. Some of my other blinds are the opposite - closed is 100%, and I wanted to maintain consistency with those blinds.

I tweaked the integration to change the "100 - position" code to just "position" which I think is what you get from Pulse 2 hub inherently, and it all works as I wanted.

Would it be possible to add an extra configuration item in addition to specifying the IP address of the hub to allow the user to choose whether they want closed to be 0% (as is the case now) or closed to be 100%. eg boolean Y/N for position inversion. Based on that flag, the code can either do the "100 -" or not.

Thanks!

**Comment by sillyfrog** [comment]
I tend to agree with you as well, however the HA spec requires 100% to be fully open, see: https://developers.home-assistant.io/docs/core/entity/cover/ - `current_cover_position`

There has been extensive discussion around this (see here, I have not read it all): https://community.home-assistant.io/t/why-the-heck-cant-we-freely-configure-a-cover-to-interpret-its-percentage-value-as-percentage-open-or-percentage-closed/225453/51

So to stick with the spec, I'm going to leave it as is, but it's easy to reverse in scripts by setting the value to `(100-x)`. 

Cheers.

**Comment by joe-sydney** [comment]
Thanks - I didn't realise that the HA specs made it that way - I had tried to look for the specs but didn't really look deep enough. I have some Tuya blinds as well, and notice that the localtuya integration (which I also use) allows for the position value to be inverted. I tried to harmonise the localtuya blinds based on the HA specs, but the slider just became non-intuitive... with 0 being open, I was able to visualise the opening and closing of the blinds with the slider position...

But nevermind... I see your integration is very stable and releases are infrequent, so I am happy to continue just modding the integration by removing the "100 -" as and when new releases are deployed.

 cheers
